packages/status-js/src/request-client/request-client.ts

```diff
@@ -127,57 +143,80 @@ class RequestClient {
     return mapUser(contactCodeAdvertisement, publicKey)
   }
 
-  private fetchCommunityDescription = async (
-    /** Uncompressed */
-    publicKey: string
-  ): Promise<CommunityDescription | undefined> => {
-    const contentTopic = idToContentTopic(publicKey)
-    const symmetricKey = await generateKeyFromPassword(publicKey)
+  public fetchCommunityDescription = async (
+    /** Compressed */
```

Comment:
Expects compressed public keys since community content topics are created from them.

Alternatively, it could accept any format and do the deserialization itself, but in cases when it's needed upfront it would have been done twice.